dev-gap/alnuth/alnuth-3.2.1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gap-pkg
+
+DESCRIPTION="Algebraic number theory and an interface to PARI/GP"
+SLOT="0"
+SRC_URI="https://github.com/gap-packages/${PN}/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+KEYWORDS="~amd64"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+# Circular dependency alnuth->polycyclic->alnuth...
+RDEPEND="dev-gap/polycyclic"
+BDEPEND="test? ( dev-gap/radiroot )"
+
+GAP_PKG_HTML_DOCDIR="htm"
+
+# The "exam" directory contains examples... but they're loaded by
+# read.g, and actually used by dev-gap/polenta!
```

Comment:
```suggestion
# read.g, and actually used by dev-gap/alnuth!
```

Reply:
Alnuth's test suite actually passes without its own examples, but the polenta test suite doesn't (without alnuth's examples). So the comment is as intended... but if some part of alnuth is using its own examples then it would be less confusing to have it say alnuth and not polenta.